pkg/diskmaker/discovery/discovery.go

```diff
@@ -260,3 +263,28 @@ func parseDeviceType(deviceType string) v1alpha1.DeviceType {
 
 	return ""
 }
+
+func bytesToTiQuantity(floatBytes float64) resource.Quantity {
+	var num int64
+	var MiB, GiB, TiB float64
+	var suffix string
+	MiB = 1024 * 1024
+	GiB = MiB * 1024
+	TiB = GiB * 1024
+
+	// Round off is required because Quantity needs a whole number.
+	// Decimal values get converted into milibytes.
+	// For example, 5.5Gi is converted into 5632Mi
```

Comment:
@gnufied Any suggestions/opinions on this rounding off behavior? (see above comment)